### ISSUE TITLE
Versioning policy for Internal modules

### ISSUE THREAD

**Issue by maoe** [haskell/pvp#8]
It's been a common practice in the Haskell community to put internal functions to `Internal` modules, which are not meant for a public API but for a semi-private API that can be useful to access library internals to extend the API. I think it's natural to relax the versioning policy for changes in those `Internal` modules. Should we mention it somehow in the PVP?

**Comment by hvr** [comment]
Yeah, I think it makes sense to formalise this, as I'd like PVP-validation tooling to be able to be aware of such that kind of convention
